### DISCUSSION TITLE
ListItem LikedBy

### DISCUSSION THREAD

**Discussion by Garabound** [microsoftgraph/msgraph-sdk-dotnet#2154] [Q&A]
I'm using the latest version 5.29.0 and I'm trying to expand the LikedBy field for SharePoint Listitem. I try the graph exlorer this endpoint
`https://graph.microsoft.com/v1.0/sites/55329914-b4ba-4df9-93a1-123123123/lists/f3e0acd5-9883-4442-b12c-123123123/items/1?expand=fields(select=LikedBy) `
Graph exlorer response with a 200 and the correct data. I check the code snipet for C#
```
var graphClient = new GraphServiceClient(requestAdapter);

var result = await graphClient.Sites["{site-id}"].Lists["{list-id}"].Items["{listItem-id}"].GetAsync((requestConfiguration) =>
{
	requestConfiguration.QueryParameters.Expand = new string []{ "fields(select=LikedBy)" };
});
```
But in the code it responde
`One or more errors occurred. (Parsing OData Select and Expand failed: Could not find a property named 'LikedBy' on type 'microsoft.graph.listItem'.)`

Can somebody help me with this simple query?
Thanks

**Comment by pschaeflein** [comment]
I'm not trying to hijack your issue, but where in the docs is "LikedBy" shown? Cause I would love to get that data.

**Reply by Garabound** [reply]
enable the rating options (like) in the library or list settings and you can access to that data

**Reply by pschaeflein** [reply]
I do not believe that to be true. The API documentation does not have "LikedBy" (at least I have not been able to find it), and the reactions has only the liked count.

**Reply by Garabound** [reply]
That is exactly what I'm asking, how I can access to that data, the rest api have that data (you can test it in graph explorer) but the c# graph library doesn't. Why?
